config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -168,7 +168,7 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: patrick.ohly@intel.com
   - name: pull-node-dra
-    test_group_name: pull-kubernetes-node-e2e-crio-dra
+    test_group_name: pull-kubernetes-node-e2e-crio-cgrpv2-dra
```

Comment:
Can we add `pull-kubernetes-node-e2e-crio-cgrpv1-dra` to this dashboard?

Reply:
updated.

Reply:
Thanks,
/lgtm